cardano-node/src/Cardano/Node/TUI/LiveView.hs

```diff
@@ -556,23 +652,26 @@ headerW p =
              $ txt (lvsNodeId p)
            ]
 
-systemStatsW :: LiveViewState a -> Widget ()
+systemStatsW :: LiveViewState blk a -> Widget ()
 systemStatsW p =
       padTop   (T.Pad 1)
     . padLeft  (T.Pad 2)
     . padRight (T.Pad 2)
-    $ vBox [ vBox [ hBox [ txt "Mempool txs:"
-                         , withAttr barValueAttr . padLeft T.Max . str . show $ lvsMempoolCapacity p
+    $ vBox [ hBox [ vBox [ hBox [ txt "Mempool (KBs):"
+                                , withAttr barValueAttr . padLeft T.Max . str .
+                                  show . (floor :: Float -> Int) . (/ 1000) . fromIntegral $ lvsMempoolCapacityBytes p
```

Comment:
KB are kilobytes? don't they need to be divided by 1024? despite that this is not correct SI unit?

Reply:
I was trying to be correct re: the unit nomenclature, yes -- (/1000) is KB, (/1024) is KiB -- which one do we want, @CodiePP?

Reply:
Going for KiB, as per private discussion.

Reply:
the file is full of "KB = 1024 bytes" assumptions. This is not SI-correct but to stay consistent within this file just use it like that.

Reply:
@CodiePP, done.